README.md.template

```diff
@@ -351,7 +351,24 @@ $ solo node start
 
 # output is similar to example-1 
 ```
+## For Developer Working on Hedera Repo
 
+To set customized `settings.txt` file, edit the file
+`./resources/templates/settings.txt` before `node setup` command.
+
+To deploy node with local build hedera code, first run `./gradlew assemble` at the root directory of hedera repo, then
+  run the following command:
+```
+$ solo node setup --local-build-path node0=<path-to-hedera-repo>,node1=<path-to-hedera-repo>,node2=<path-to-hedera-repo>
+```
+To have nodes running with different versions of hedera application, please clone the repo in multiple directories,
```

Comment:
Need to specify which repo. I think you meant the `hedera-services` repo, right?
I think you need to add instructions a bit chronologically assuming a Hedera core developer is setting things up. So I would recommend we add steps like
   1. Clone repo
   2. Build
   3. Update settings / config files
   4. Run node setup...

Reply:
Readme template updated